tools/Dockerfile-tty

```diff
@@ -4,35 +4,73 @@
 # Using ubuntu:noble base image
 FROM ubuntu:jammy
 
+# Version definitions and checksums
+ARG TARGETARCH
```

Comment:
I am not sure about it. This ARG suggests that the build could be multi arch but after close look most fetched binaries are amd64. 

I would stick to only amd64 architecture for time being.